checker/src/main/java/org/checkerframework/checker/signedness/SignednessUtilExtra.java

```diff
@@ -0,0 +1,63 @@
+package org.checkerframework.checker.signedness;
+
+import java.awt.Dimension;
+import java.awt.image.BufferedImage;
+import org.checkerframework.checker.signedness.qual.Unsigned;
+
+/**
+ * Provides more static utility methods for unsigned values. These methods use Java packages not
+ * included in Android. {@link SignednessUtil} has more methods.
```

Comment:
I would include more of the text from the manual, to make the motivation for this class clear.

Reply:
Or, reference the manual.  That way avoids text duplication and the possibility that the two versions get out of sync.